Source/DafnyLanguageServer.Test/Synchronization/DiagnosticsTest.cs

```diff
@@ -594,19 +594,6 @@ decreases y
       Assert.Contains("a postcondition", diagnostics[0].Message);
     }
 
-    [Fact]
-    public async Task OpeningDocumentThatIncludesDocumentWithSemanticErrorsReportsResolverErrorAtInclude() {
-      var source = "include \"syntaxError.dfy\"";
```

Comment:
It was a duplicate of `OpeningDocumentThatIncludesDocumentWithSyntaxErrorsReportsParserErrorAtInclude`

Also the title was wrong, since it uses the file `syntaxError` and the title contains `SemanticErrors`.